samza-core/src/test/java/org/apache/samza/execution/TestExecutionPlanner.java

```diff
@@ -65,26 +65,7 @@
   private StreamSpec output1;
   private StreamSpec output2;
 
-  private JoinFunction createJoin() {
-    return new JoinFunction() {
-      @Override
-      public Object apply(Object message, Object otherMessage) {
-        return null;
-      }
-
-      @Override
-      public Object getFirstKey(Object message) {
-        return null;
-      }
-
-      @Override
-      public Object getSecondKey(Object message) {
-        return null;
-      }
-    };
-  }
-
-  private SystemAdmin createSystemAdmin(Map<String, Integer> streamToPartitions) {
+  static SystemAdmin createSystemAdmin(Map<String, Integer> streamToPartitions) {
 
     return new SystemAdmin() {
```

Comment:
Would still prefer using mocks. Makes the expectations in tests much more clearer.